pkgs/by-name/su/surrealist/package.nix

```diff
@@ -18,12 +19,30 @@
 , rustPlatform
 , stdenv
 , stdenvNoCC
-, webkitgtk
+, webkitgtk_4_1
 }:
 
 let
 
-  esbuild-20-2 = let version = "0.20.2";
+  cargo-tauri_2 = let
+    version = "2.0.0-rc.3";
+    src = fetchFromGitHub {
+      owner = "tauri-apps";
+      repo = "tauri";
+      rev = "tauri-v${version}";
+      hash = "sha256-PV8m/MzYgbY4Hv71dZrqVbrxmxrwFfOAraLJIaQk6FQ=";
+    };
+  in cargo-tauri.overrideAttrs (drv: {
+    inherit src version;
+    cargoDeps = drv.cargoDeps.overrideAttrs (lib.const {
+      inherit src;
+      name = "tauri-${version}-vendor.tar.gz";
+      outputHash = "sha256-BrIH0JkGMp68O+4B+0g7X3lSdNSPXo+otlBgslCzPZE=";
+    });
+  });
```

Comment:
Why isn't this a top level package? If this package needs it, others might too, right?

Reply:
Definitely makes sense! :+1: But it might needs to be coordinated with https://github.com/NixOS/nixpkgs/pull/335751